lib/Error.hs

```diff
@@ -37,7 +37,7 @@ instance Buildable InternalCommandsError where
   build = \case
     InvalidEntry entry ->
```

Comment:
Also rename this variable from `entry` to `pathSegment` while we're here.

Reply:
Ok!